pkg/dwarf/op/regs.go

```diff
@@ -52,10 +80,22 @@ func (regs *DwarfRegisters) Bytes(idx uint64) []byte {
 
 // Reg returns register idx or nil if the register is not defined.
 func (regs *DwarfRegisters) Reg(idx uint64) *DwarfRegister {
-	if idx >= uint64(len(regs.Regs)) {
-		return nil
+	if idx >= uint64(len(regs.regs)) {
+		if regs.loadMoreCallback != nil {
+			regs.loadMoreCallback()
```

Comment:
I think we should put this into a `regs.loadMore` method which can wrap the logic of checking if the callback is not nil and then setting it to nil once it's been called.